src/status_im/contexts/chat/messenger/messages/content/view.cljs

```diff
@@ -199,10 +201,13 @@
                [content.text/text-content message-data context]
 
                constants/content-type-emoji
-               [not-implemented/not-implemented [old-message/emoji message-data]]
+               [emoji-message/view
+                (-> message-data
```

Comment:
sorry, neatpick :)
because is the chat view, we might want to squeeze as much perf as possible, so maybe best to just create a new object instead of `select-keys` + `assoc` i.e

```
{:content (:content message-data)
 :last-in-group? (:last-in..... messag-data)
...
:in-pinned-view? (:in-pinned-view? context)}

```

just looking at a very rough perf difference:
```
user=> (defn extract-data-2 [m] {:a (:a m) :b (:b m) :c (:c m) :g "g"})
#'user/extract-data-2
user=> (defn extract-data-1 [m] (-> m (select-keys [:a :b :c]) (assoc :g "g")))
#'user/extract-data-1
user=> (def message-data {:a "a" :b "b" :c "c" :d "d" :e "e" :f "f"})
#'user/message-data
user=> (time (dotimes [_ 10000] (extract-data-1 message-data)))
"Elapsed time: 5.786936 msecs"
nil
user=> (time (dotimes [_ 100000] (extract-data-1 message-data)))
"Elapsed time: 87.96445 msecs"
nil
user=> (time (dotimes [_ 1000000] (extract-data-1 message-data)))
"Elapsed time: 735.203584 msecs"
nil
user=> (time (dotimes [_ 1000000] (extract-data-1 message-data)))
"Elapsed time: 848.839143 msecs"
nil
user=> (time (dotimes [_ 1000000] (extract-data-1 message-data)))
"Elapsed time: 757.222673 msecs"
nil
user=> (time (dotimes [_ 1000000] (extract-data-1 message-data)))
"Elapsed time: 717.327672 msecs"
nil
user=> (time (dotimes [_ 1000000] (extract-data-2 message-data)))
"Elapsed time: 109.881926 msecs"
nil
user=> (time (dotimes [_ 1000000] (extract-data-2 message-data)))
"Elapsed time: 89.065761 msecs"
nil
user=> (time (dotimes [_ 1000000] (extract-data-2 message-data)))
"Elapsed time: 162.977585 msecs"
nil
user=> (time (dotimes [_ 1000000] (extract-data-2 message-data)))
"Elapsed time: 171.493888 msecs"
nil
user=> (time (dotimes [_ 1000000] (extract-data-2 message-data)))
"Elapsed time: 118.874594 msecs"
nil
user=> (time (dotimes [_ 1000000] (extract-data-2 message-data)))
"Elapsed time: 111.5382 msecs"
```

avoiding select-keys+assoc is 6/7 times faster, and it doesn't (imo) impact readability so much

Reply:
thanks for pointing out. Didn't consider this :+1: